e2e/browser-runner/lit.test.js

```diff
@@ -122,6 +122,27 @@ describe('Lit Component testing', () => {
             expect(await $('div*=me').getHTML(false)).toBe('Find me')
         })
 
+        it('fetches the parent element by content correctly', async () => {
+            render(
+                html`<button><span>Click Me!</span></button>`,
+                document.body
+            )
+            expect(await $('span=Click Me!').getHTML(false)).toBe('Click Me!')
+            expect(await $('button=Click Me!').getHTML(false)).toBe('<span>Click Me!</span>')
+            const elem = $('button=Click Me!')
+            await expect(elem).toHaveText('Click Me!')
+        })
+
+        it('fetches the element with multiple text nodes by the content', async () => {
+            render(
+                html`<p>
+                Find
+                me</p>`, document.body
+            )
```

Comment:
I am not sure of Lit renders this as two text nodes, so let's do this:
```js
const container = document.createElement('p')
container.appendChild(document.createTextNode('Find'))
container.appendChild(document.createTextNode('me'))
document.body.appendChild(container)
```

Reply:
sure